pkgs/by-name/ig/ignition/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  gettext,
+  desktop-file-utils,
+  appstream,
+  blueprint-compiler,
+  pkg-config,
+  gjs,
+  gtk4,
+  wrapGAppsHook4,
+  libadwaita,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ignition";
+  version = "1.1.3";
+
+  src = fetchFromGitHub {
+    owner = "flattool";
+    repo = "ignition";
+    tag = finalAttrs.version;
+    hash = "sha256-XVBlwonMHb78XF6mpPYLJ68E5Tb+dFVFqNSsVCCS0xc=";
+  };
+
+  patches = [
+    # hack to make gjs properly find our binary
+    ./fix-effective-entrypoint-path.patch
+  ];
+
+  postPatch = ''
+    # upstream uses gjs from the build platform instead of the host platform
+    # it's not very important, as I don't even know if cross compilation works with GTK
+    substituteInPlace src/meson.build \
+      --replace-fail "find_program('gjs').full_path()" "'${lib.getExe gjs}'"
+  '';
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+
+    appstream
+    blueprint-compiler
+    desktop-file-utils
+    gettext
+    gtk4
```

Comment:
I think `gtk` can move to `buildInputs`

Reply:
Using `wrapGAppsHook4` adds `gtk4` to both `buildInputs` and `nativeBuildInputs`.
Meaning we could just as well remove `gtk4` from the `nativeBuildInputs`.

Should I do it?